package-lock.json

```diff
@@ -19794,7 +19794,7 @@
 				},
 				"nopt": {
 					"version": "4.0.1",
-					"resolved": false,
+					"resolved": "https://registry.npmjs.org/nopt/-/nopt-4.0.1.tgz",
```

Comment:
I believe the package-lock change is probably not needed in this PR, right?

Reply:
> I believe the package-lock change is probably not needed in this PR, right?

yeah, right, this was made automatically, by "npm install" I guess.